blogweb/simpletree/index.rst

```diff
@@ -0,0 +1,45 @@
+SimpleTree Code Sprint

```

Comment:
It's not a code sprint so I would change the name here; maybe SimpleTree project fits better?